CMakeLists.txt

```diff
@@ -1199,6 +1204,14 @@ if(BUILD_TESTING)
   add_subdirectory(tests)
 endif()
 
+# NTLM support requires crypto function adaptions from various SSL libs
+# TODO alternative SSL libs tests for SSP1, GNUTLS, NSS
+if(NOT CURL_DISABLE_CRYPTO_AUTH AND (USE_OPENSSL OR USE_WINDOWS_SSPI OR USE_DARWINSSL OR USE_MBEDTLS OR USE_WIN32_CRYPTO))
```

Comment:
This is missing NSS and SECTRANSP (what I'm pretty sure replaces DARWINSSL). Is there a reason for that?

Reply:
No, that's a mistake.